src/prefect/server/database/interface.py

```diff
@@ -402,7 +402,7 @@ async def get_flow_run_notifications_from_queue(
             session=session, limit=limit
         )
 
-    async def read_configuration_value(self, session: sa.orm.Session, key: str):
+    async def read_configuration_value(self, session: AsyncSession, key: str):
```

Comment:
We could probably just do a full-blown `s/sa.orm.Session/AsyncSession/g` through the whole repo because there's nowhere that we're actually using a sync session to me knowledge.  I guess the imports will also need to be fixed up too, but I wonder how pervasive it is now

Reply:
There's only 10 left now, I can follow this up and replace those.